### PR TITLE
test(pod/pill/portrait): apply new url approach

### PR DESCRIPTION
**Pull request by KatarzynaQA**
### Proposed behaviour

Apply new `url` approach in tests for:
`pod` (timing: was `2:25` -> `00:47`),
`pill` (timing: was  `1:15` -> `00:29` ),
`portrait` (timing  `1:36` -> `00:19`).

### Current behaviour

Currently we are passing all parameters via `knobs`.

### Checklist

- [ ] Screenshots are included in the PR
- [ ] Carbon implementation and Design System documentation are congruent
- [ ] All themes are supported
- [x] Commits follow our style guide
- [ ] Unit tests added or updated
- [x] Cypress automation tests added or updated
- [ ] Storybook added or updated
- [ ] Typescript `d.ts` file added or updated

### CODE REVIEW

File: cypress/features/regression/pod.feature [hunk 1]
```diff
@@ -1,146 +1,133 @@
 Feature: Pod component
-  I want to test Pod component
-
-  Background: Open Pod component page
-    Given I open "Pod" component page
+  I want to check Pod component
```
Comment (DlgSHi) [inline cypress/features/regression/pod.feature:2]:
```suggestion
  I want to test Pod component
```

File: cypress/features/regression/pod.feature [hunk 2]
```diff
@@ -1,146 +1,133 @@
...
     Then Edit property is visible
 
-  @positive
-  Scenario: Enable onEdit checkbox and check the edit event
-    Given I check onEdit checkbox
-      And clear all actions in Actions Tab
-    When I click onEdit icon
-    Then edit action was called in Actions Tab
-
   # width is properly for CI
```
Comment (DlgSHi) [inline cypress/features/regression/pod.feature:105]: do we need this comment ?

File: cypress/features/regression/portrait.feature [hunk 3]
```diff
@@ -1,17 +1,14 @@
 Feature: Portrait default component
-	I want to test Portrait default component
-
-	Background: Open Portrait default component page
-		Given I open "Portrait" component page
+	I want to check Portrait default component
```
Comment (DlgSHi) [inline cypress/features/regression/portrait.feature:2]:
```suggestion
	I want to test Portrait default component
```

File: cypress/features/regression/portrait.feature [hunk 4]
```diff
@@ -1,17 +1,14 @@
 Feature: Portrait default component
```
Comment (DlgSHi) [inline cypress/features/regression/portrait.feature:1]: Why indents are so big ?

File: cypress/features/regression/portrait.feature [hunk 5]
```diff
@@ -42,12 +39,8 @@ Feature: Portrait default component
 
 	@positive
 	Scenario Outline: Set Portrait src to <source>
-		Given I select source to "src"
-		When I set src to "<source>"
+		When I open default "Portrait" component in noIFrame with "portrait" json from "commonComponents" using "src" object name
 		Then Portrait src value is set to "<source>"
```
Comment (DlgSHi) [inline cypress/features/regression/portrait.feature:43]: how you are passing this `source` value here ?

Comment (KatarzynaQA) [reply cypress/features/regression/portrait.feature]: fixed

File: cypress/features/regression/portrait.feature (hunk 4 above)
Comment (KatarzynaQA) [reply cypress/features/regression/portrait.feature:1]: I don't know. On my local it's ok

File: cypress/features/regression/portrait.feature (hunk 3 above)
Comment (KatarzynaQA) [reply cypress/features/regression/portrait.feature]: fixed

File: cypress/features/regression/pod.feature (hunk 2 above)
Comment (KatarzynaQA) [reply cypress/features/regression/pod.feature]: fixed

File: cypress/features/regression/pod.feature (hunk 1 above)
Comment (KatarzynaQA) [reply cypress/features/regression/pod.feature]: fixed

**Pull request merged**